conf/routes

```diff
@@ -10,19 +10,29 @@ GET /unsupported-browser                            controllers.Application.unsu
 
 GET  /*path/  controllers.Application.removeTrailingSlash(path: String)
 
-# ----- Bundles Landing Page ----- #
+# ----- Sitemap ----- #
+
+GET /sitemap.xml                                    controllers.SiteMap.sitemap
 
-GET /uk                                             controllers.Application.supportLanding()
+
+# ----- Geo landings ----- #
 GET /us                                             controllers.Application.redirect(location="/us/contribute")
 GET /au                                             controllers.Application.redirect(location="/au/contribute")
 GET /eu                                             controllers.Application.redirect(location="/eu/contribute")
 GET /int                                            controllers.Application.redirect(location="/int/contribute")
 GET /nz                                             controllers.Application.redirect(location="/nz/contribute")
 GET /ca                                             controllers.Application.redirect(location="/ca/contribute")
+GET /uk                                             controllers.Application.redirect(location="/uk/support")
 GET /                                               controllers.Application.geoRedirect()
-GET /showcase                                       controllers.Application.showcase()
 
-GET /sitemap.xml                                    controllers.SiteMap.sitemap
+
+# ----- Bundles Landing Page ----- #
+
+GET /support                                        controllers.Application.redirect(location="/uk/support")
```

Comment:
Wait a second, this will redirect everyone, globally, to the UK showcase page. Do we need to something different for US etc?

Reply:
I just made up `/support` for this PR, the goal for our team is to eventually have internationalised showcase pages but right now we are starting with UK. Afaik the rest of redirects (/) and (/{country}) will preserve their rules